sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetFileFormat.scala

```diff
@@ -49,6 +49,15 @@ object GpuParquetFileFormat {
 
     val parquetOptions = new ParquetOptions(options, sqlConf)
 
+    val columnEncryption = options.getOrElse("parquet.encryption.column.keys", "")
+    val footerEncryption = options.getOrElse("parquet.encryption.footer.key", "")
+
+    if (!columnEncryption.isEmpty || !footerEncryption.isEmpty) {
+      meta.willNotWorkOnGpu("Encryption is not yet supported on GPU. To enable unset the " +
```

Comment:
"To enable" is misleading, as this will not enable encryption.  I suggest wording it as "If encrypted Parquet reads or writes are not required..." to be more explicit.